src/vs/workbench/contrib/terminal/browser/terminalView.ts

```diff
@@ -81,11 +80,6 @@ export class TerminalViewPane extends ViewPane {
 	) {
 		super(options, keybindingService, _contextMenuService, _configurationService, _contextKeyService, viewDescriptorService, _instantiationService, openerService, themeService, telemetryService);
 		this._register(this._terminalService.onDidRegisterProcessSupport(() => {
-			if (this._actions) {
```

Comment:
@Tyriar As far as I could tell, this property is unused

Reply:
Looks like we removed the usage and forgot to clean up 👍